sim/dex.js

```diff
@@ -374,6 +374,9 @@ class ModdedDex {
 			// Inherit any statuses from the base species (Arceus, Silvally).
 			const baseSpeciesStatuses = this.data.Statuses[toId(template.baseSpecies)];
 			if (baseSpeciesStatuses !== undefined) {
+				// don't overwrite name/id with the generic ones from the status
+				delete baseSpeciesStatuses.id;
+				delete baseSpeciesStatuses.name;
```

Comment:
You should not be mutating this...

Reply:
...I'm deleting them from the status itself, ain't I

What's the best way to do this the, put previous id/name in a temp variable and reassign later? Use a for-in loop to copy everything except name and id?

Reply:
I would probably use a for-in loop to copy only things the current thing doesn't have.

Reply:
Hm if I just use a normal for-in then typescript complains about ``template[key] = ...`` because `Element implicitly has an 'any' type because type 'Template' has no index signature.`

Reply:
Just `ts-ignore`.